src/pages/AdminPage/AdminCircleModal.tsx

```diff
@@ -218,7 +219,8 @@ export const AdminCircleModal = ({
         allocText !== circle.allocText ||
         vouchingText !== circle.vouchingText ||
         onlyGiverVouch !== circle.only_giver_vouch ||
-        teamSelection !== circle.team_selection
+        teamSelection !== circle.team_selection ||
```

Comment:
@f1lander I'm not understanding how "Team Selection Enabled" is tied to "Auto Opt Out". Can you explain that to me more?

Reply:
The giant compound condition this line is in just means "have any form fields changed". So this line has changed just because of the addition of the line below.